tests/MongoDB.Driver.Legacy.Tests/MongoServerSettingsTests.cs

```diff
@@ -611,20 +633,20 @@ public void TestRetryWrites()
         }
 
         [Fact]
-        public void TestLocalThreshold()
+        public void TestScheme()
```

Comment:
Weird diff

Reply:
Sorry. TestLocalThreshold was out of order and I moved it.

Perhaps I should not have.